infra/terraform/openai.tf

```diff
@@ -20,8 +20,8 @@ resource "azurerm_cognitive_deployment" "gpt" {
     version = var.openai_model_version
   }
 
-  scale {
-    type     = "Standard"
+  sku {
```

Comment:
The azurerm provider is currently pinned to 3.113.0 (see main.tf) and this change will cause terraform validation errors